jdbc-adapter/virtualschema-jdbc-adapter/pom.xml

```diff
@@ -14,6 +14,17 @@
         <profile>
             <id>it</id>
             <dependencies>
+                <dependency>
+                    <scope>verify</scope>
+                    <groupId>org.postgresql</groupId>
+                    <artifactId>postgresql</artifactId>
+                    <version>42.2.5</version>
+                </dependency>
+                <dependency>
+                    <groupId>org.apache.hive</groupId>
```

Comment:
Which scope is this?

Reply:
This is the profile *IT* which is used in the integration test. Before this was a general dependency which makes no sense.